llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -6465,12 +6465,12 @@ bool SPIRVToLLVM::transMetadata() {
           }
         }
 
-          // Give the workgroup size to the middle-end.
```

Comment:
Please look at the same place in the internal branch before making this change.

Reply:
@JaxLinAMD  I've checked, but there's some mismatch, please tell me whether this change is needed.

Reply:
I've tried to submit without this commit, but the clang-format check failed. It seems i had to include this commit as far as lib/SPIRV/SPIRVReader.cpp is being modified.

Reply:
I don't understand why you got a clang-format build failure without this change. You are not otherwise editing this area, so git-clang-format should not be looking at it.

Making this change will cause Jax problems with cherry-picking to the internal branch, which will get resolved back to the internal branch stage, and this branch will end up changing back next time internal changes are pushed back out here.